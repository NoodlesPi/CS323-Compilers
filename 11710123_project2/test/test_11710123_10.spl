struct a{
    int b;
    int c;
    int d;
};

int calculate_total_time(struct a app){
    return app.b * 30 + app.c - 60;
}

int main(){
    struct a app1;
    int n = 1;
    int arr[10][20];
    char c_arr[5];
    n = n && 1 || c_arr[3]; 
    arr[2][2] = c_arr[1];
    return calculate_total_time(app1);
}
