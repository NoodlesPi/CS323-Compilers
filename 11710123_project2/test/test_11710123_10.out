Error type 7 at Line 8: unmatching operands, such as adding an integer to a structure variable
Error type 13 at Line 8: accessing member of non-structure variable
Error type 7 at Line 8: unmatching operands, such as adding an integer to a structure variable
Error type 8 at Line 8: the functionâ€™s return value type mismatches the declared type
Error type 7 at Line 16: unmatching operands, such as adding an integer to a structure variable
Error type 5 at Line 16: unmatching types on both sides of assignment operator (=)
Error type 5 at Line 17: unmatching types on both sides of assignment operator (=)
